site/source/index.md

```diff
@@ -31,8 +31,6 @@ Here are two Spacebars templates from an example app called "Leaderboard" which
   <ol class="leaderboard">
     {{#each players}}
       {{> player}}
-    {{else}}
```

Comment:
No need to do that. You can update the branch itself and it will update the pull request.

An opportunity for you to learn git more. :-)